### ISSUE TITLE
what is the correct command line to run test data?

### ISSUE THREAD

**Issue by binzhougithub** [MouseLand/cellpose#208]
Thank you for sharing this tool. Your method is fantastic.  For example,  the cellpose’s performance is much better then Mask-RCNN on generalized data,  i.e.  AP=0.94 on cellpose  vs AP=0.72 on Mask-RCNN.  After downloading the software and dataset, I ran this command line,

python -m cellpose --dir  ~/data/test --pretrained_model cyto --chan 2 --chan2 1 --diameter 0 --save_png  --use_gpu

Some results are amazing, their AP0.5 are close to 1. However I got a couple of low AP0.5 cases,  I guess something wrong with my command line. Maybe I didn’t turn on tiling or TTA.  I also noticed that there are 68 images in downloaded test dataset, but there are only 57 images in the paper’s test dataset. I guess you removed some non-cell images. AP0.5 was calculated in my own program,  is there option to output this AP0.5 in cellpose?

Regards
Bin

**Comment by binzhougithub** [comment]
I read the source code of prediction part.  I found the default tile_overlap is 0.1 in source code,  I changed it to 0.5 to match the value of literature.  the AP0.5 becomes better a bit now.  the average AP is 0.79 for all 68 images, still not close to 0.94.   

Could you let me know what mistakes I made?  how do I specify  flow_threshold(default=0.4) and cellprob_threshold(default=0.0)?

Thanks
Bin

**Comment by carsen-stringer** [comment]
You may be referring to the APs for the specialized data, those are the first 11 images in the training set.

Here are the divisions of images 11 and up into different categories: https://github.com/MouseLand/cellpose/blob/3ac46b2f1fdad9ad955c95446414c9ba31cbb391/paper/performance_figs.py#L516
